### ISSUE TITLE
50fps video incorrectly reported as 60fps

### ISSUE THREAD

**Issue by riking** [rg3/youtube-dl#4911]
Reproduction:

![image](https://cloud.githubusercontent.com/assets/627891/6101887/f549fab8-afe0-11e4-9912-8442928e8cb3.png)

```
$ youtube-dl -F https://www.youtube.com/watch?v=26B-XArAeeM
[youtube] 26B-XArAeeM: Downloading webpage
[youtube] 26B-XArAeeM: Extracting video information
[youtube] 26B-XArAeeM: Downloading DASH manifest
[info] Available formats for 26B-XArAeeM:
format code extension resolution  note 
140         m4a       audio only  DASH audio  145k , audio@128k (44100Hz), 1.84MiB (worst)
141         m4a       audio only  DASH audio  274k , audio@256k (44100Hz), 3.65MiB
160         mp4       256x144     DASH video  121k , 13fps, video only, 1.33MiB
133         mp4       426x240     DASH video  306k , 25fps, video only, 2.92MiB
134         mp4       640x360     DASH video  400k , 25fps, video only, 3.01MiB
135         mp4       854x480     DASH video  887k , 25fps, video only, 6.17MiB
136         mp4       1280x720    DASH video 1847k , 25fps, video only, 12.17MiB
298         mp4       1280x720    DASH video 2654k , h264, 60fps, video only, 14.86MiB
17          3gp       176x144     
36          3gp       320x240     
5           flv       400x240     
43          webm      640x360     
18          mp4       640x360     
22          mp4       1280x720    (best)
```

**Comment by jaimeMF** [comment]
Thanks for the report! it should be fixed in the next version.

**Comment by phihag** [comment]
Fixed in yotuube-dl 2015.09.02.3 and newer. See [our FAQ](https://yt-dl.org/update) if you need help updating.
